types/api/reward.ts

```diff
@@ -1,4 +1,4 @@
 export interface Reward {
   reward: string;
-  type: 'Miner Reward' | 'Validator Reward' | 'Emission Reward' | 'Chore Reward' | 'Uncle Reward' | 'POA Mania Reward';
+  type: 'Miner Reward' | 'Validator Reward' | 'Emission Reward' | 'Chore Reward' | 'POA Mania Reward';
```

Comment:
Probably it is outside the scope of this PR but i think we can also remove the POA reward as it part of the Gnosis ecosytem only. Don't know it there are reference to this on the current front end but if so I will remove it

Reply:
Thanks Davide, I'll make a note of it and keep it in mind if we ever see any references to POA in our project. With the current env variables, it shouldn't, but definitely something to take a look on. Thank you!